### PR TITLE
Update QUEST.tsv [Lines 1951-2100]

### PR DESCRIPTION
**Pull request by ghost**
Lines 1951-2100 reviewed. Lines 1954 and 2006 are poems that I'll leave alone until we start localization or for a translator to do. Open to feedback.

### CODE REVIEW

**Comment by laurencee** [comment]
Have you tested those unicode ellipsis characters in the actual client? 
One of the unicode ellipsis I tried during the last beta displayed vertically centered.

**Comment by ghost** [comment]
As far as I know, I'm using the one that displays correctly, but haven't tested it myself. If it doesn't work during the next CBT, we can just do an automated search and replace.

**Comment by laurencee** [comment]
Sounds reasonable to me. :+1:

File: QUEST.tsv [hunk 1]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150428_001962	$Anyone entering here unauthorized will be punished for their sin.
+QUEST_20150428_001963	$The Order to Move Out
+QUEST_20150428_001964	$This facility will soon be shut down. {nl}I ask all construction workers and related people of the royal tomb to move out.
+QUEST_20150428_001965	$The Diary of Baron Soldier, Chagos
+QUEST_20150428_001966	$The magician, who appeared from somewhere, told the ignorant baron that the object is located here.
+QUEST_20150428_001967	$The way to eternal youth. {nl}I can't find the way there: the place where only death awaits.
+QUEST_20150428_001968	$The Crumbling Document
+QUEST_20150428_001969	$The demon lord, Helgasercle, divided the spirit of the demon lord, Haubuck. {nl}This was used to power the barrier that now surrounds the Mage Tower and then was scattered throughout the kingdom.
+QUEST_20150428_001970	$Many magicians came to break the barrier, but they could not succeed with just the foundations stones that were gathered at the tower.
+QUEST_20150428_001971	$To all adventurers of the kingdom, {nl}the Mage Tower offers gold to those who bring back the pieces of Haubuck's spirit.
```
Comment (Soukyuu) [inline QUEST.tsv]: "pieces" -> "fragments" seems to fit better when talking of spirit/soul

File: QUEST.tsv [hunk 2]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150428_001972	$The Order to Defeat the Demons
+QUEST_20150428_001973	$I hereby order in the name of Vakarine! {nl}After Medis Diena, they can no longer be forgiven. {nl}They are trying to expand their power by penetrating into the crack that was created in the barrier.
+QUEST_20150428_001974	$All the Cupols here shall defeat demon lords Boolut, Nuaelle, and Baltrus first. {nl}The enemy will panic without their commanders and will be easier to defeat.
+QUEST_20150428_001975	$Cupols who can't attack should concentrate on blocking the crack. {nl}We can't allow more demons from going to the demon lords.
+QUEST_20150428_001976	$How should I live from now on? {nl}I should have moved to Klaipeda.
+QUEST_20150428_001977	$Since you've placed them into the inventory, you can read them anytime.
+QUEST_20150428_001978	$The power of Mirtis was so overwhelming that Cunningham had to make an important decision.
+QUEST_20150428_001979	$I hear the more and more monsters are appearing in the Baron Arleruno's land. {nl}Please defeat those monsters before they take any innocent lives.
```
Comment (Soukyuu) [inline QUEST.tsv]: Should be "I hear **that** more and more monsters are appearing in the Baron Arleruno's land. "

File: QUEST.tsv [hunk 3]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150428_001980	$According to the information I've gathered from the wings of Baibora, the demons come out in groups from a place called Demon's Prison. {nl}Please help us with this to prevent a greater chaos from happening.
+QUEST_20150428_001981	$Help the druids that were dispatched to the farm of the Green Family. {nl}They're looking for people who can help with the mass purification.
+QUEST_20150428_001982	$Dina Bee Farm is in danger of monster attacks. {nl}They can't even work so they desperately need help.
+QUEST_20150511_001983	$It's better to use the cable car than to head straight into the monsters.
```
Comment (Soukyuu) [inline QUEST.tsv]: "head straight into the monsters" sounds weird. I think the meaning here is that it's better to use the cable car than having to fight monsters on your way to the same destination. How about 
"It's better to use the cable car than having to fight your way past monsters."?

File: QUEST.tsv [hunk 4]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001984	$Just don't believe everything he says. {nl}He was known for being quite the show-off back at home.
+QUEST_20150511_001985	$It would be nice if all these Revelators being here meant that the Goddesses are back.
+QUEST_20150511_001986	$Power is the most important attribute for swordsman. {nl}You could also train yourself to be agile and possess strong stamina. {nl}You should first think about what to become from now on.
```
Comment (Soukyuu) [inline QUEST.tsv]: Maybe wording it as "what to concentrate/prioritize from now on" is better than "what to become from now on", because no matter which you choose, you will stay a swordsman. Same for the other lines using the same formulation.

File: QUEST.tsv [hunk 5]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001987	$Power is the most important attribute for archers. {nl}But, of course you can't ignore agile movement. {nl}You should first think about what to become from now on.
+QUEST_20150511_001988	$Intelligence is the most important attribute for Clerics. {nl}But sometimes you many need strong power or spirit. {nl}You should first think about what to become from now on. 
+QUEST_20150511_001989	$Helping reconstruct Fedimian may be more meaningful than pottery. {nl}But people need something to soothe their hearts.
+QUEST_20150511_001990	$If I had the diary, I could continue my adventure, but I wasn't selected… {nl}But if it were you…
+QUEST_20150511_001991	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
```
Comment (Soukyuu) [inline QUEST.tsv]: The cold (air) is uncountable, so your version is talking about the common cold instead: look [here](http://www.macmillandictionary.com/dictionary/british/cold_2)

Should reword it to "If you cleverly use the dry air that forms around areas of severe cold, you'd be able to..."

File: QUEST.tsv [hunk 6]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001992	$I'll report the damages. {nl}Chasing a Sparnas, a gigantic bird-type creature, will be hard. Despite being large, they are rather agile. {nl}It usually goes after the supplies, but I can't imagine nor find out why. I'm guessing someone's playing a trick.
+QUEST_20150511_001993	$Grain pocket was discovered in the ashes of the supplies. {nl}They smell so bad and old they can't be used for food at all. {nl}Please check the supplies for anything missing.
+QUEST_20150511_001994	$Defeat all the monsters nearby. {nl}The vicious energy in the vine forest is constantly creating new monsters. {nl}But you should first defeat most of them before you face Ironbaum. {nl}Even a small factor could ruin the big task. {nl}Also, you could try threatening the monsters so they can't do what they want to do.
+QUEST_20150511_001995	$(Everything is scribbled so poorly that they can't be distinguished from letters or drawings.)
```
Comment (Soukyuu) [inline QUEST.tsv:1995]: This kind of loses the original meaning for me.
"(everything is scribbled so poorly, one can't distinguish the letters from drawings.)"

File: QUEST.tsv [hunk 7]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001996	$The facility that belonged to the workers who worked on the Mausoleum of Zachariel suddenly shut down. {nl}It's not like they left because it was useless but they all just vanished all at once. {nl}Maybe it is related to the irregularly big size of the tomb and its complicated composition {nl}even after considering the fact that it was the grave to commemorate the greatest king in history.
```
Comment (Soukyuu) [inline QUEST.tsv]: It's not quite clear what the "it" useless refers to was. From the text it seems they're talking about finishing the thing, so you should change that to "It's not like they left because it was hopeless to finish it, but they all just vanished all at once." Maybe let @SHUcream00/@Requinn look at the line?

File: QUEST.tsv [hunk 8]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001997	$King Zachariel had left many arrangements for future generations. {nl}We and the Jonas family were one of them. {nl}But the meaning of it is only known by the Goddess. {nl}At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.
```
Comment (Soukyuu) [inline QUEST.tsv]: > King Zachariel had left many arrangements 

Wouldn't "King Zachariel had made many arrangements" be better?

> But the meaning of it is only known by the Goddess.

If this "it" refers to the arrangements, then it should be changed to something like:
"But their meaning is only known by the Godess."
Or, to make it less confusing:
"But the meaning of the arrangements is only known by the Godess."

>  At least we know that the will of the king that we are protecting and the secret of the tomb indicates one person.

The meaning of this line eludes me. Seems machine translated.

File: QUEST.tsv [hunk 9]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001998	$Trees become bigger as years pass. {nl}We can only see Woodman here but if you think of the large trees or the trees that stand in the old capital city… {nl}You can't help but think of the existence of even larger trees. {nl}After the disaster occurred, many plants turned into monsters so we can't ever rule out that possibility.
```
Comment (Soukyuu) [inline QUEST.tsv]: The Woodman thing seems weird (at least I have no idea what it's supposed to be) and makes the rest of the sentence weird. Let's try re-translating?

File: QUEST.tsv [hunk 10]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_001999	$(There are barely legible words written here.) {nl}Euros could not hold his breath any more so he ended up inhaling the spores and died. {nl}Mushwort hunt people with the cruelest method I've ever seen. {nl}They walks around our gathering places and shakes out the mushrooms that would release spores.
```
Comment (Soukyuu) [inline QUEST.tsv]: If Mushwort is a monster name, then it should be "Mushworts" here. Also: "They walks around our gathering places and shakes out", should be "They walk" and "shake out".

File: QUEST.tsv [hunk 11]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002000	$We never know when it will release spores. If we don't stop it, the whole camp could be annihilated. {nl}I have no way to keep safe from it. I can't stop coughing…
```
Comment (Soukyuu) [inline QUEST.tsv]: Now it's singular again. Please make it uniform

File: QUEST.tsv [hunk 12]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002003	$For the first step: recharge the magical essence I gave to you. Look for where magic is condensed at Pavydo Mixed Forest. {nl}You won't be able to use the magical essence right away, however, since it has also absorbed some malign energy. {nl}For that, you'll need to collect Operor saliva and purify it at the altar of purification at the Wasteland of Renistu.
+QUEST_20150511_002004	$Name: High Raven{nl}Sex: Male{nl}Occupation: Merchant{nl}List of items: luxurious goods, weapons {nl}{nl}In accordance with kingdom laws, you may enter.
+QUEST_20150511_002005	$If you're seeing this you either didn't pay me in full or did something I told you not to do. {nl}I have cursed this letter. {nl}I've no idea what's gonna happen to you from now on, but know that my curse is absolute.
+QUEST_20150511_002006	$The one who fell while investigating the knowledge. {nl}Return to the Goddess by becoming a part of the canyon. {nl}The saddened mind breaks the gold stones of the canyon and the poured alcohol soaks into the ground on behalf of the tears.
```
Comment (Soukyuu) [inline QUEST.tsv:2006]: So that's one of the poems, eh.
I'm not quite sure, but isn't if "on behalf of some**one**"? I think you should use "instead of tears" here.

File: QUEST.tsv [hunk 13]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002007	$Eventually, Ruklys had failed. {nl}He was destroyed and broken by those who were jealous of him. {nl}He left the Goddess in my hands. {nl}
+QUEST_20150511_002008	$Every object possess the inner strength of its possessor. {nl}And to add to the magical interpretations on that, they could also transform the shapes of their possessors. {nl}Objects belonging to demons could do it with relative ease. (The same applies to the sacred areas, but don't even dare doing that in those places.)
+QUEST_20150511_002009	$But that's still just a theory despite the efforts of many frontiers. {nl}The possessor's inner strength and interpretations can never be fully understood if it were not for the possessor himself. {nl}That's why we hope for the blessing from the Goddess or the wisdom from the possessor for the knowledge we lack.
+QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to seek help from others. {nl}We can't be deceived any more. {nl}We have to protest against the violence behind the mask that claims to protect us.
+QUEST_20150511_002011	$We believe your help is the main factor to our success so that our efforts were not made in vain.
```
Comment (Soukyuu) [inline QUEST.tsv]: The tense seems wrong here. 
"so that our efforts were not made in vain" -> "so that our efforts will not end up being in vain"

File: QUEST.tsv [hunk 14]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002012	$We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to make it. {nl}I believe only this place can answer the question I've had in my mind for a long time.
```
Comment (Soukyuu) [inline QUEST.tsv]: "to make it" -> "to build it"? "To make a tomb" sounds weird.

File: QUEST.tsv [hunk 15]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002014	$There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002015	$With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
+QUEST_20150511_002016	$Some poles unlock the tomb and some altars lock it. {nl}The magical energy that surrounds the tomb moves by the key. {nl}But none can be that key. {nl}We are not chosen.
+QUEST_20150511_002017	$The curse disturbs the rules of the universe that are being controlled by the Goddess with human greed.{nl}These rules in the human world get influenced by the personalities of each human, their abilities and their potential power. {nl}The cases when these abilities get released depend on the conditions above.
```
Comment (Soukyuu) [inline QUEST.tsv]: "The curse disturbs the rules of the universe that are being controlled by the Goddess **through** human greed.{nl}These rules in the human world get influenced by the personalities of each **person**, their abilities and their potential power."

For the first one, I think it's "the curse disturbs the rules through human greed", not "the rules are controlled with human greed". For the second one, I think using "person" instead of "human" in this case is more natural.

File: QUEST.tsv [hunk 16]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002018	$According to the old writing that is left on the great cathedral, that is using a small part of the Goddess' abilities secretly.{nl}Some people say that after seducing weak minds from humans, the demons used them for the source of the curse. {nl}May the Goddess bless!
```
Comment (Soukyuu) [inline QUEST.tsv]: > According to the old writing that is left on the great cathedral, that is using a small part of the Goddess' abilities secretly.

what is "that" in this context?

> Some people say that after seducing weak minds from humans, the demons used them for the source of the curse. 

"weak minds from humans" > "weak minds of people"

> May the Goddess bless!

Bless who/what?

File: QUEST.tsv [hunk 17]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002019	$There are two ways to release the curse: {nl}Remove the curse using a greater power or to eliminate the caster of the curse. {nl}
```
Comment (Soukyuu) [inline QUEST.tsv]: > Remove the curse using a greater power or to eliminate the caster of the curse.

should either use the "to <verb>" form for both or drop it from both:
"To remove the curse using a greater power or to eliminate the caster of the curse."

File: QUEST.tsv [hunk 18]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002020	$Removing the curse with a greater power: {nl}This way depends on whether the power that overwhelms the curse comes from good or evil and from darkness or light.{nl}{nl}The most important factor for eliminating the caster: {nl}Even after eliminating the caster, the curse may not be removed due to other factors.{nl}{nl}I have no doubt that the power of the Goddess is stronger.
```
Comment (Soukyuu) [inline QUEST.tsv]: should be "using a greater power" to keep uniformity with the previous sentence.

> This way depends on whether the power that overwhelms the curse comes from good or evil and from darkness or light.

Is good/light and evil/darkness different things in ToS universe? The whole enumeration part sounds awkward. Maybe checking the translation is a good idea?

File: QUEST.tsv [hunk 19]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002021	$There are mainly two ways to spread the curse. {nl}First, using equipment and uttering simple words. It sounds easy but it is, in fact, the hardest way. {nl}Therefore, from the example of the Pilgrim Path, we mainly use the cursed ones as the mediums to spread the curse.
+QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous influence on the mind of the victim. Their ego usually can't endure the feeling of guilt this causes. {nl}It seems the magic that was used on the cursed city of Ruklys seems to be of a higher level than the one used on Naktis. We need to look into it more.
```
Comment (Soukyuu) [inline QUEST.tsv]: > It seems the magic that was used on the cursed city of Ruklys seems to be of a higher level than the one used on Naktis. We need to look into it more.

Redundant "seems".

File: QUEST.tsv [hunk 20]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002023	$The Completed Research Materials of Varkis
```
Comment (Soukyuu) [inline QUEST.tsv]: Complete?

File: QUEST.tsv [hunk 21]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002024	$The First Research Material{nl}-{nl}We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to make it.{nl}I believe only this place can answer the question I've had in my mind for a long time.
```
Comment (Soukyuu) [inline QUEST.tsv]: These lines seems to duplicate some of the above lines. Please see my comments on those lines and apply them here as well.

File: QUEST.tsv [hunk 22]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150511_002029	$A wish would come true if we make a pot and put the wish in it. {nl}I understand that it's tradition, but for now, restoring Fedimian back to its former glory is more important than that.
+QUEST_20150511_002030	$I can't guarantee the safety of other holy relics or the keys so you better hurry.
+QUEST_20150511_002031	$I'm just listening! I'm not going to do what you say! {nl}I don't think wheat will grow here no matter what kind of magic you use. {nl}I need to look for my son's body, so you better keep it short!
+QUEST_20150511_002032	$The picture of the essence is drawn on the old surface.
+QUEST_20150511_002033	$The Eternal Adventure
+QUEST_20150714_002034	$Well, you're a new face. Did Julian send you?
+QUEST_20150714_002035	$Yeah. There's something weird about Goddess Laima. {nl}No one seems to know when she disappeared. To be honest nobody seems to really know whether or not she actually is a Goddess.
+QUEST_20150714_002036	$She's never appeared before. {nl}Although I've read somewhere that she does appear every now and then…
```
Comment (Soukyuu) [inline QUEST.tsv]: Let's not use the 's contraction when we mean "has", it's not a good style.

File: QUEST.tsv [hunk 23]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002039	$When all is said and done, I don't lie about my prices. Everything I sell is priced honestly and accordingly.
+QUEST_20150714_002040	$Fire is a grace sent to us by the Goddess Gabija. {nl} You won't become a great Pyromancer if you don't acknowledge that.
+QUEST_20150714_002041	$The Goddesses may be all gone but my powers are still doing fine. {nl} I'm sure it's proof that Goddess Gabija is at least safe somewhere.
+QUEST_20150714_002042	$Killing enemies one by one from a distance is the beauty of the bow and arrow. {nl}It doesn't matter how many shots you have to take, the shots that actually hit {nl}are the ones that count.
+QUEST_20150714_002043	$Monk Lotze
+QUEST_20150714_002044	$I'll do whatever it takes to take back the monastery from the monsters.
+QUEST_20150714_002045	$I don't like those hunters. {nl}Outsiders must not interfere with the affairs of the monastery.
+QUEST_20150714_002046	$Monk Jidas
+QUEST_20150714_002047	$It would do us good if everyone would stopped being so stubborn. {nl}I'm sure the hunters will be a great help in retaking the monastery…
```
Comment (Soukyuu) [inline QUEST.tsv]: > It would do us good if everyone would stopped being so stubborn. 

Is the "would" here an expression of [politeness](http://www.englisch-hilfen.de/en/grammar/will_would_if_clauses.htm)? If yes, then "would stopped" -> "would stop". 

Otherwise it's not valid to use "would" here, "would stopped" -> "stopped"

File: QUEST.tsv [hunk 24]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002058	$Rumor has it that Ruklys has heard about the citizens' harsh conditions {nl}and is negotiating with Kadumel for their amnesty. {nl}If the general wants, we could just stand still…
+QUEST_20150714_002059	$Premier Eminent is in talks with General Ruklys. {nl}We may have to move to the Great Earth Fortress and let the royal forces claim the city.
+QUEST_20150714_002060	$It's good news for our the civilians at least. Despite the town being conceded, the Fortress will not be taken as easily.
+QUEST_20150714_002061	$King Kadumel is keen on persecuting the townspeople and burning the Great Earth Fortress.{nl} The premier is pushing General Ruklys to negotiate terms ensuring that the civilians are safe from enemy forces.
+QUEST_20150714_002062	$Despite the royal forces occupying the town, there has yet to be any threatening movement. {nl}What's going to happen to us…
+QUEST_20150714_002063	$As promised, I will cast magic on you that will protect you from Petrified Frosts. {nl}But if you move out from this region, this magic won't be as effective so be careful.
```
Comment (Soukyuu) [inline QUEST.tsv]: > But if you move out from this region

But if you leave this region

File: QUEST.tsv [hunk 25]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002064	$There has been no progress since Ruklys' group retreated to the Great Earth Fortress. {nl}The priests under Maven's direction are reluctant to fight, and the King is hiring mages by paying them a hefty sum.
+QUEST_20150714_002065	$But we expect little to no progress at all even after their participation.
+QUEST_20150714_002066	$How could this happen?! {nl}Lydia Schaffen is aiding Kadumel's forces!
+QUEST_20150714_002067	$Lydia Schaffen, a person I've only heard of in rumors, came to the city with her followers. {nl}The officers and soldiers expect she'll change the tide of the battle. {nl}His Majesty ordered us to support her to the best of our ability.
+QUEST_20150714_002068	$General Ruklys has fallen in battle, and the Great Earth Fortress has been taken. {nl}Some people say the general died to Lydia Schaffen's arrow, and others say he died to the royal magicians' magic. {nl}It is indeed sad news.
```
Comment (Soukyuu) [inline QUEST.tsv]: > Some people say the general died to Lydia Schaffen's arrow, and others say he died to the royal magicians' magic. 

died by?

File: QUEST.tsv [hunk 26]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002069	$Propaganda of the Kingdom
+QUEST_20150714_002070	$Surrender by throwing away your weapons.{nl}It is indeed your fault that you've created the Great Earth Fortress to stand against the kingdom, but merciful King Kadumel will save you.
```
Comment (Soukyuu) [inline QUEST.tsv]: > merciful King Kadumel will save you.

merciful King Kadumel will spare you?

File: QUEST.tsv [hunk 27]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002073	$Even Lydia Schaffen, friend of Ruklys, turned to the side of kingdom soldiers. {nl}No one will stand on your side. {nl}With your hands, collapse The Great Earth Fortress, the proof of your treason, and wave the white flag. {nl}Faithful Premier Eminent shall forgive all.
+QUEST_20150714_002074	$Are you going to die with Ruklys who misled you with malicious tricks to create The Great Earth Fortress to stand against the kingdom? {nl}Or live a new life by serving the king until the end by following the way of Premier Eminent who protected the kingdom from the rebellion?
+QUEST_20150714_002075	$The Records of Rebel Forces
+QUEST_20150714_002076	$As I was retreating from kingdom soldiers, I encountered many demons. {nl}It doesn't make sense. Where did they come from?
+QUEST_20150714_002077	$There's no hope at all. Ah… my lovely Ridell. {nl}The kingdom soldiers did not keep their word. We're all going to die.
+QUEST_20150714_002078	$Those kingdom soldiers! Especially that Premier Eminent! They're no different from demons. {nl}Even the Goddesses would not forgive you.
```
Comment (Soukyuu) [inline QUEST.tsv]: > Even the Goddesses would not forgive you.

Maybe this should be "Even the Goddesses would not forgive them."? Them = soldiers and the minister

File: QUEST.tsv [hunk 28]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002079	$I don't want to die. {nl}We were just following the the Goddesses' words. {nl}Everyone is killing each other while in the names of the Goddesses. {nl}Who has deceived to us? {nl}
```
Comment (Soukyuu) [inline QUEST.tsv]: What is this I don't even xD
Sorry, your version doesn't make any sense. It should be something like
"I don't want to die. {nl}We were just following the words of the Goddesses.{nl}Everyone is killing each other in the name of the Goddesses.{nl}Who has deceived us?{nl}

File: QUEST.tsv [hunk 29]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002080	$Kingdom Military Recruit Propaganda
+QUEST_20150714_002081	$Kingdom Military needs competent people like you! {nl}Loyalty to the kingdom! To victory and a brighter future!
+QUEST_20150714_002082	$Swear loyalty to Kingdom Military and become a new person. {nl}The Goddesses protect our armies.
+QUEST_20150714_002083	$Life and death… which one would you choose? {nl}The only way to survive is to swear loyalty to the kingdom!
+QUEST_20150714_002084	$Don't forget that we won 600 years ago. {nl}Join the military. You sins can be forgiven.
+QUEST_20150714_002085	$Please move with us if possible. {nl}There's been a number of monster sightings near here.
+QUEST_20150714_002086	$The supplies should be here soon… {nl}Are they going to be late again?
+QUEST_20150714_002087	$You will need various tools when travelling. {nl}It's better to buy them in advance.
+QUEST_20150714_002088	$Goddess Laima… Actually, I don't know her that well. No one does, except for the disciples. {nl}I heard she's the Goddess of Destiny and Forecasts, but she has never shown up.
```
Comment (Soukyuu) [inline QUEST.tsv]: Which disciples are meant here? If it's "disciples of Laima", then that should be changed to "except for her disciples".
The part about her not showing up doesn't make much sense.

File: QUEST.tsv [hunk 30]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002089	$I just remembered something while I was talking to you. {nl}Maybe Goddess Laima knew everything about the disaster beforehand.
+QUEST_20150714_002090	$I pray to Goddess Gabija everyday. It keeps the house warm. {nl}As you may already know, Goddess Gabija controls fire and since I'm a housewife, I am close to fire as well.
+QUEST_20150714_002091	$If you cleverly use the dry air that forms around severe colds, you'd be able to freeze any object even in a desert. {nl}However, if you can't receive the blessing from the Goddess, it won't be possible to make that kind of air.
```
Comment (Soukyuu) [inline QUEST.tsv]: change line 1191 to this! seems to be the same, but doesn't have the issue you introduced through your edit in line 1191

File: QUEST.tsv [hunk 31]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002092	$The Records of the Master Himself
+QUEST_20150714_002093	$I've studied magic for a long time as a sorcerer. {nl}From the process, the advancement of sorcerer can't be explained without relating with demons, and since they've fought against the Goddesses, the field has been widened to the study of the Goddesses as well.
```
Comment (Soukyuu) [inline QUEST.tsv]: > From the process, the advancement of sorcerer can't be explained without relating with demons, and since they've fought against the Goddesses, the field has been widened to the study of the Goddesses as well.

"sorcerer" should probably be "Sorcery"

File: QUEST.tsv [hunk 32]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150714_002094	$While I was doing that, I learned of the field of accumulation. {nl}It's an empty field. The theory of the beginning and the end is a fantastic library at the boundary of transcendency, all for one purpose.
+QUEST_20150714_002095	$The Records about Disciple Raius
+QUEST_20150714_002096	$My other disciple, Raius, is clever and a fast learner but has an insatiable greed for knowledge. I am worried that he may ruin the task. {nl}A scholar should always try to learn, but obsessing about it will make one walk the wrong path.
```
Comment (Soukyuu) [inline QUEST.tsv]: > I am worried that he may ruin the task.

I am worried that he may fail the task.

File: QUEST.tsv [hunk 33]
```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
...
+QUEST_20150428_001955	$The sons of the guards rest here.
+QUEST_20150428_001956	$The Grass of Zachariel King
+QUEST_20150428_001957	$The king told those who engraved the letters and the symbols onto the hard woods {nl}to abstain from sin, but they lost their memories as the king ordered. {nl}I did it because I was afraid that the demons and the people with malicious intent may steal my artifacts. {nl}However, I don't want to repeat that for my people.
+QUEST_20150428_001958	$(The book has rotted so much that nothing inside can be read.)
+QUEST_20150428_001959	$The minister said, {nl}"We already heard the words of His Majesty. {nl}We don't know exactly why, but the reason this royal tomb was built was not to show off, {nl}but to prepare for the forecast that may bring chaos to the kingdom and change the future of the humans. {nl}If we don't do this important task, who will?
+QUEST_20150428_001960	$Warning Sign for Unauthorized Entrants
+QUEST_20150428_001961	$To protect the sacred secret of Zachariel King, no one may enter this facility.
```
Comment (Soukyuu) [inline QUEST.tsv]: Missed that one yesterday: should change to "the sacred secret of King Zachariel"

**Comment by ghost** [comment]
I'd say this is good to merge. We can just have a translator take a look at the lines in the future when one is available.

**Comment by Soukyuu** [comment]
I'd avoid marking those with `$` in the meantime, but otherwise, good to merge.

**Comment by imcgames** [comment]
Thank you :smile:
